cosmo_tester/test_suites/image_based_tests/agent_installation_test.py

```diff
@@ -335,26 +317,30 @@ def _install_script(name, windows, user, manager, attributes, tmpdir, logger):
         constants.REST_PORT_KEY: str(defaults.INTERNAL_REST_PORT),
         constants.BROKER_SSL_CERT_PATH: local_cert_path,
         constants.LOCAL_REST_CERT_FILE_KEY: local_cert_path,
-        constants.MANAGER_FILE_SERVER_URL_KEY:
+        constants.MANAGER_FILE_SERVER_URL_KEY: (
             'https://{0}:{1}/resources'.format(manager.private_ip_address,
-                                               defaults.INTERNAL_REST_PORT),
-        constants.MANAGER_FILE_SERVER_ROOT_KEY: str(tmpdir)
+                                               defaults.INTERNAL_REST_PORT)
+        ),
+        constants.MANAGER_FILE_SERVER_ROOT_KEY: str(tmpdir),
     }
     (tmpdir / 'cloudify_agent').mkdir()
 
     ctx = MockCloudifyContext(
-            node_id='node',
-            tenant={'name': 'default_tenant'},
-            rest_token=manager.client.tokens.get().value,
-            properties={'agent_config': {
-                'user': user,
-                'windows': windows,
-                'install_method': 'init_script',
-                'rest_host': manager.private_ip_address,
-                'broker_ip': manager.private_ip_address,
-                'name': name
-            }})
-
+        node_id='node',
+        tenant={'name': 'default_tenant'},
+        properties={'agent_config': {
+            'user': user,
+            'windows': windows,
+            'install_method': 'init_script',
+            'rest_host': manager.private_ip_address,
+            'broker_ip': manager.private_ip_address,
+            'name': name
+        }})
+    internal_ctx_dict = getattr(ctx, '_context')
+    internal_ctx_dict.update({
```

Comment:
Why the move back to using this hack? What was wrong with providing the tenant and the token as arguments to `MockCloudifyContext`?

Reply:
Oh, that might work- that modification was one I was having to merge in and I must've missed what that other block of code was doing. I'll try it without this.

Reply:
Yeah, it should work. This was done a few days ago and the tests seem to pass with it